src/lib/HederaLib.ts

```diff
@@ -68,4 +68,14 @@ export class HederaWallet {
       return { error: (e as Error).message }
     }
   }
+
+  public signMessage(bytes: string) {
+    const buf = Buffer.from(bytes, 'base64')
```

Comment:
@hgraphql I'm not sure if the `sign` method was automatically converting the string that was being passed to it, but it's function signature does not indicate that it would do so. `Buffer` is a subclass of `Uint8Array`, so this conversion should work as expected. I compared the signatures generated from this `buf` variable, and after creating a `Uint8Array` instance (`const arr = new Uint8Array(Buffer.from(bytes, 'base64'))`). The signatures were identical, so it seems like just the conversion to buffer is sufficient

Reply:
<img width="749" alt="Screenshot 2023-08-04 at 9 41 10 AM" src="https://github.com/hgraph-io/hedera-walletconnect-wallet/assets/136644362/8aa38b46-24f5-48d4-b8b5-12488e7ebf85">
<img width="421" alt="Screenshot 2023-08-04 at 9 40 59 AM" src="https://github.com/hgraph-io/hedera-walletconnect-wallet/assets/136644362/acb3bf18-9f1e-4717-b01d-95176c4ae184">